test/unit/resources/firewalld_test.rb

```diff
@@ -0,0 +1,50 @@
+# encoding: utf-8
+# author: Matthew Dromazos
+
+require 'helper'
+require 'inspec/resource'
+
+describe 'Inspec::Resources::FirewallD' do
+  resource = load_resource('firewalld')
+  centResource = MockLoader.new(:centos7).load_resource('firewalld')
+
+  it 'verify firewalld detects a zone' do
+    centResource.stubs(:has_zone?).with('public').returns(true)
```

Comment:
This line, and line 13, are not actual tests but rather stubs/mocks. Please add tests.